modelzoo/rnnt/pytorch/rnnt.patch

```diff
@@ -3771,15 +3797,16 @@ index 0000000..758849c
 +
 +MODEL_ARGS="--enc_n_hid 1024 \
 +  --enc_pre_rnn_layers 2 \
-+  --enc_stack_time_factor 8 \
++  --enc_stack_time_factor 2 \
```

Comment:
please explain this change

Reply:
it is model hyper parameter for RNN-T, default `8` result poor converge, change to `2` for default value which can converge to 5.8% WER.

Reply:
Got it, WER as 5.8% just for original RNNT not optimized one right?

Reply:
yes, optimized WER is 6.97%